src/lib/orionTypes/EntityTypeVectorResponse.cpp

```diff
@@ -90,13 +90,13 @@ std::string EntityTypesResponse::check
 
 /* ****************************************************************************
 *
-* EntityTypesResponse::present -
+* EntityTypeVectorResponse::present -
 */
-void EntityTypesResponse::present(const std::string& indent)
+void EntityTypeVectorResponse::present(const std::string& indent)
 {
-  LM_F(("%s%d EntityTypesResponses:\n", indent.c_str(), typeEntityVector.size()));
+  LM_F(("%s%d EntityTypeVectorResponse:\n", indent.c_str(), entityTypeVector.size()));
```

Comment:
EntityTypeVectorResponse => EntityTypeResponses.

Or even, to be clearer: %d items in entityTypeVector/EntityTypeVectorResponse ...